docs/diginstroom/sip/2.0/profiles/film.md

```diff
@@ -0,0 +1,251 @@
+---
+layout:       default
+title:        Film
+parent:       Profiles
+grand_parent:  2.0
+nav_order:    4
+nav_exclude:  false
+---
+Editor's Draft
+{: .label .label-yellow }
+# Profile: Film 
+
+The film profile supports the ingest of digitised film consisting of one or more image and/or audio reels. 
+This profile dictates how the media files (in file formats such as ZIP, MOV and DPX), their metadata, and the relationships between them, should be expressed and organized.
+
+It mainly applies the [DCTERMS metadata schema](https://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd) for descriptive metadata and allows extensions using [Schema.org](https://schema.org), thereby resembling the [Material Artwork profile](https://data.hetarchief.be/id/sip/2.0/material-artwork). 
```

Comment:
From 1.2 on, the material artwork profile and the basic profile are aligned in terms of descriptive metadata, so  it's better to refer to the basic profile (that's what material artwork does)